src/components/select-field/props.js

```diff
@@ -3,6 +3,9 @@
 export type SelectOptionValueProp = string | number | null;
 
 export type SelectOptionProp = {
+    avatarUrl?: string,
     displayText: string,
+    hasWarning?: boolean,
+    id?: string | number,
```

Comment:
`string | number | null`

Reply:
i couldn't do null as a test said it was an incompatible type with rendering the PillComponent